packages/react-sdk/docusaurus/docs/React/03-ui-components/core/call-layout.mdx

```diff
@@ -3,14 +3,95 @@ id: call-layout
+    </StreamVideo>
+  );
+};
+```
+
+## Customization
+
+Both [built-in layouts](#built-in-layouts) allow for some customization of the layout via props. However, it's also possible to create your own layout, see our [Custom Call Layout guide](../../../ui-cookbook/custom-call-layout) for more information.
 
 ## Built-in Layouts
 
 ### `PaginatedGridLayout`
 
+#### Props
+
+| Name                      | Description                                                                                            | Type                                                                                                   |
+| ------------------------- | ------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------ |
+| `groupSize`               | The number of participants to display per page                                                         | `number` \| `undefined`                                                                                |
+| `excludeLocalParticipant` | Whether to exclude the local participant from the grid                                                 | `boolean` \| `undefined`                                                                               |
+| `pageArrowsVisible`       | Turns on/off the pagination arrows                                                                     | `boolean` \| `undefined`                                                                               |
+| `ParticipantViewUI`       | [See `ParticipantView` documentation](../../../ui-components/core/participant-view/#participantviewui) | [See `ParticipantView` documentation](../../../ui-components/core/participant-view/#participantviewui) |
+| `VideoPlaceholder`        | [See `ParticipantView` documentation](../../../ui-components/core/participant-view/#videoplaceholder)  | [See `ParticipantView` documentation](../../../ui-components/core/participant-view/#videoplaceholder)  |
+
 ### `SpeakerLayout`
+
+#### Props
+
+| Name                         | Description                                                                                                                                                | Type                                                                                                   |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `participantsBarPosition`    | The position of the participants who are not in focus, the default is `bottom`                                                                             | `top` \| `bottom` \| `left` \| `right`                                                                 |
```

Comment:
```suggestion
| `participantsBarPosition`    | The position of the participants who are not in focus, the default is `bottom`                                                                             | `top` \| `bottom` \| `left` \| `right` \| null                                                                |
```

Will land in the next PR - `null` means to not render the participants bar.

Reply:
👍 I'll add the explanation as well

Reply:
Thanks!